garage/experiment/local_tf_runner.py

```diff
@@ -171,11 +179,11 @@ def obtain_samples(self, itr, batch_size):
             One batch of samples.
 
         """
-        if self.n_epoch_cycles == 1:
+        if self.train_args["n_epoch_cycles"] == 1:
```

Comment:
referencing a dict like this is hacky-looking. why not just store it in `self.n_epoch_cycles`?

Reply:
I am grouping arguments for `train()` in a dict, so that it can be returned by `restore()` as a whole.   But true, if I do it this way, it is not so clear what are included in `train_args` by just reading the code.

Reply:
please use the more explicit style. `types.SimpleNameSpace` might be helpful here.